module/product/src/Application/Controller/Api/Attribute/UpdateProductAttributeAction.php

```diff
@@ -0,0 +1,144 @@
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+use Ergonode\Core\Domain\Query\LanguageQueryInterface;
+
+class UpdateProductAttributeAction
+{
+    private CommandBusInterface $commandBus;
+
+    private AttributeValueConstraintProvider $provider;
+
+    private ValidatorInterface $validator;
+
+    private LanguageQueryInterface $query;
+
+    public function __construct(
+        CommandBusInterface $commandBus,
+        AttributeValueConstraintProvider $provider,
+        ValidatorInterface $validator,
+        LanguageQueryInterface $query
+    ) {
+        $this->commandBus = $commandBus;
+        $this->provider = $provider;
+        $this->validator = $validator;
+        $this->query = $query;
+    }
+
+    /**
+     * @Route(
+     *     name="ergonode_product_attribute_update",
+     *     path="products/{product}/attribute/{attribute}",
+     *     methods={"POST"},
+     *     requirements={
+     *         "product"="[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}",
+     *         "attribute"="[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"
+     *     }
+     * )
+     * @Route(
+     *     "/products/{product}/draft/{attribute}/value",
+     *     methods={"PUT"},
+     *     requirements = {
+     *        "product" = "[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}",
+     *        "attribute" = "[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"
+     *     }
+     * )
+     *
+     * @IsGranted("PRODUCT_UPDATE")
+     * @IsGranted("edit", subject="language")
+     *
+     * @SWG\Tag(name="Product")
+     * @SWG\Parameter(
+     *     name="language",
+     *     in="path",
+     *     type="string",
+     *     required=true,
+     *     default="en_GB",
+     *     description="Language Code",
+     * )
+     * @SWG\Parameter(
+     *     name="product",
+     *     in="path",
+     *     type="string",
+     *     description="Product id",
+     * )
+     * @SWG\Parameter(
+     *     name="attribute",
+     *     in="path",
+     *     type="string",
+     *     description="Attribute id",
+     * )
+     * @SWG\Parameter(
+     *     name="value",
+     *     in="formData",
+     *     type="string",
+     *     description="Attribute value",
+     *     required=true,
+     * )
+     * @SWG\Response(
+     *     response=200,
+     *     description="Change product attribute Value",
+     * )
+     * @SWG\Response(
+     *     response=400,
+     *     description="Validation error",
+     *     @SWG\Schema(ref="#/definitions/validation_error_response")
+     * )
+     *
+     * @throws \Exception
+     */
+    public function changeDraftAttribute(
+        Language $language,
+        AbstractProduct $product,
+        AbstractAttribute $attribute,
+        Request $request
+    ): Response {
+        $value = $request->request->get('value');
+        $value = $value !== '' ? $value : null;
```

Comment:
```suggestion
        $value = $value === '' ? null : $value;
```
It's easier to read with no negation so if not a problem please use it like this